### PR TITLE
Use IRSA for metamorph

### PR DESCRIPTION
**Pull request by taal-martins-eglitis**
Use the default config via IRSA, update the Dockerfile, use context passed by args

### CODE REVIEW

File: metamorph/store/dynamodb/dynamodb.go [hunk 1]
```diff
@@ -28,24 +26,11 @@ type DynamoDB struct {
 const ISO8601 = "2006-01-02T15:04:05.999Z"
 
 func New() (store.MetamorphStore, error) {
-	// export necessary parameters for aws dynamodb connection
-	err := os.Setenv("AWS_ACCESS_KEY_ID", viper.GetString("metamorph.db.dynamodb.awsAccessKeyID"))
```
Comment (boecklim) [inline metamorph/store/dynamodb/dynamodb.go:32]: If we don't need this, we can also remove it from the default config file: https://github.com/bitcoin-sv/arc/blob/c64f83a63a82bc0a446661b929d522abc14be4d6/config.yaml#L50

Comment (taal-martins-eglitis) [reply metamorph/store/dynamodb/dynamodb.go:32]: Yes, indeed. Will update.

File: Dockerfile [hunk 2]
```diff
@@ -1,5 +1,7 @@
 FROM ubuntu:latest
 
+RUN apt-get update && apt-get install ca-certificates -y && apt-get clean && rm -rf /var/lib/apt/lists/*
```
Comment (boecklim) [inline Dockerfile:3]: Possibly we can do this using this package like this: https://github.com/breml/rootcerts
```go
import (
    _ "github.com/breml/rootcerts"
)
```

But we can also do that in a separate PR

**Review by boecklim — APPROVED** [review]
